inc/namespace.php

```diff
@@ -267,6 +273,18 @@ function add_login_logo() {
 	<?php
 }
 
+/**
+ * Add the favicon-icon when the site has no icon set.
+ */
+function add_favicon_icon() {
+	if( has_site_icon() ){
+		return;
+	}
+
+	$favicon = get_site_icon_url( '', Altis\get_config()['modules']['cms']['favicon-icon'] );
+	echo '<link rel="icon" href="' . esc_url( $favicon ) . '" />';
+}
```

Comment:
@shadyvb I included a check here for when the `has_site_icon` and returns if it's present before continuing for use the filter you suggested.

Reply:
I'd get rid of `add_favicon_icon` altogether and just use the filter to let WordPress do its markup output. Sorry if i wasn't clear on that.